src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java

```diff
@@ -129,6 +129,14 @@ public void accettazioneDelegaPF() {
         deleghePagoPATest.siControllaCheLaDelegaHaLoStatoAttiva(nomeFilePersonaFisica);
     }
 
+    public void accettaDelegaPF() {
+        deleghePagoPATest.waitDelegheButton();
+        deleghePagoPATest.siSceglieOpzioneAccetta();
+        deleghePagoPATest.siInserisceIlCodiceDelegaNelPopUp();
+        deleghePagoPATest.siCliccaSulBottoneAccetta();
+        deleghePagoPATest.siControllaCheLaDelegaHaLoStatoAttiva(nomeFilePersonaFisica);
+    }
```

Comment:
There is already this action right before this one

Reply:
here the delegation is accepted with api code while in the others it is not and transforming it would have cost too much time for a single test